src/test/java/wolox/training/controllers/BookControllerTest.java

```diff
@@ -150,4 +152,32 @@ public void updateBadId() throws Exception {
                 .andExpect(status().is4xxClientError());
 
     }
+
+    @Test public void findNotFound() throws Exception {
+        mvc.perform(get("/api/books/search?isbn=sarasa"))
+                .andExpect(status().is4xxClientError());
+    }
+
+    @Test public void findByIsbnFoundOpenLibrary() throws Exception {
+        String isbnInOpenLibrary = "0385472579";
+        mvc.perform(get("/api/books/search?isbn=" + isbnInOpenLibrary))
+                .andExpect(status().isCreated());
+    }
+
+    @Test public void findByIsbnExists() throws Exception {
+        aBook = new Book();
+        aBook.setTitle("title test");
+        aBook.setGenre("genre test");
+        aBook.setAuthor("author test");
+        aBook.setImage("image test");
+        aBook.setIsbn("isbn sarasa");
+        aBook.setPages(4321);
+        aBook.setSubtitle("subtitle test");
+        aBook.setPublisher("publisher test");
+        aBook.setYear("year test");
+
+        given(service.findByIsbn(aBook.getIsbn())).willReturn(aBook);
+        mvc.perform(get("/api/books/search?isbn=" + aBook.getIsbn()))
+                .andExpect(status().isCreated());
```

Comment:
If I'm not mistaken, this should be a status 200 OK because the book already existed in the database. Also, you could check the JSON it answers against the content of the mocked book.